templates/controllers/tab/settings/productionStage/form/productionStageForm.tpl

```diff
@@ -46,4 +44,3 @@
 		{fbvFormButtons id="productionStageFormSubmit" submitText="common.save" hideCancel=true}
 	{/if}
 </form>
-<p><span class="formRequired">{translate key="common.requiredField"}</span></p>
```

Comment:
maybe. I think we can follow what the php form enforces. If it says it is not required, let's just get rid of those required statements on the form template and move on.